src/workers/backgroundWorkerService.ts

```diff
@@ -150,121 +159,24 @@ async function runCleanupWorkerSafely(
 		const stats = await cleanupOldInstances(drizzleClient, logger);
 
 		const duration = Date.now() - startTime;
-		logger.info("Cleanup worker completed successfully", {
-			duration: `${duration}ms`,
-			organizationsProcessed: stats.organizationsProcessed,
-			instancesDeleted: stats.instancesDeleted,
-			errorsEncountered: stats.errorsEncountered,
-		});
+		logger.info(
+			{
+				duration: `${duration}ms`,
+				organizationsProcessed: stats.organizationsProcessed,
+				instancesDeleted: stats.instancesDeleted,
+				errorsEncountered: stats.errorsEncountered,
+			},
+			"Cleanup worker completed successfully",
+		);
 	} catch (error) {
 		const duration = Date.now() - startTime;
-		logger.error("Cleanup worker failed", {
-			duration: `${duration}ms`,
-			error: error instanceof Error ? error.message : "Unknown error",
-			stack: error instanceof Error ? error.stack : undefined,
-		});
-	}
-}
-
-/**
```

Comment:
Why are these lines from No. 169 to No. 249 deleted when the comments state why they should remain?